pipelines/build/common/config_regeneration.groovy

```diff
@@ -26,6 +26,7 @@ class Regeneration implements Serializable {
     private final String javaVersion
     private final Map<String, Map<String, ?>> buildConfigurations
     private final Map<String, ?> targetConfigurations
+    private final String activeNodeTimeout
```

Comment:
Can be removed if we're removing if we're not specifying a node timeout in the regen jobs

Reply:
Sorted.